### PR TITLE
node conversion

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

**Comment by dickeyxxx** [comment]
can you review @tt?

File: commands/download.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const co = require('co')
+
+function * run (context) {
+  const download = require('../lib/download')
+  const repo = require('../lib/repo')
+  const app = context.app
+
+  let url = yield repo.getURL(app)
+  let filename = context.args.filename || `${app}-repo.tgz`
+  console.error(`Downloading slug to ${filename}`)
```
Comment (tt) [inline commands/download.js]: "repository"

Comment (dickeyxxx) [inline commands/download.js]: heh yep, copy paste issue

File: commands/gc.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+curl -o /dev/null --upload-file ../repack.tgz '${yield repo.putURL(app)}'
+exit`
+  })
+  yield dyno.start()
+}
+
+module.exports = {
+  topic: 'repo',
+  command: 'gc',
+  description: 'run a git gc --agressive on the applications repository',
```
Comment (tt) [inline commands/gc.js]: "application's"

Comment (tt) [inline commands/gc.js]: "aggressive"

Comment (tt) [inline commands/gc.js]: Sorry, I see this was how it was before.

**Comment by tt** [comment]
Looks good to me.

File: commands/gc.js (hunk 2 above)
Comment (dickeyxxx) [inline commands/gc.js]: fixed. That was actually a typo in the old code too.

**Pull request merged**